corehq/apps/accounting/static/accounting/js/confirm_plan.js

```diff
@@ -57,25 +57,30 @@ hqDefine('accounting/js/confirm_plan', [
             }
         };
         self.submitDowngrade = function (pricingTable, e) {
-            var finish = function () {
-                if (self.form) {
-                    self.form.submit();
-                }
-            };
-
             var $button = $(e.currentTarget);
             $button.disableButton();
-            $.ajax({
-                method: "POST",
-                url: initialPageData.reverse('email_on_downgrade'),
-                data: {
-                    old_plan: self.currentPlan.name,
-                    new_plan: self.newPlan.name,
-                    note: $button.closest(".modal").find("textarea").val(),
-                },
-                success: finish,
-                error: finish,
-            });
+            if (self.form) {
+                var downgradeReason = "";
+                var selectedDowngradeOptions = document.getElementById("select-downgrade-reason").selectedOptions;
```

Comment:
Style nitpick: it'd be more concise to do something like $("#select-downgrade-reason").val().join(", ")